src/core/utils.spec.ts

```diff
@@ -4,12 +4,12 @@ import { formatAddress, parseAddress, isEmptyAddress, computeTransactionFee } fr
 import { Wei } from './entities'
 
 describe('Formatting address', () => {
-  const ADDRESS = parseAddress('0xae3a8a1D877a446b22249D8676AFeB16F056B44e')
+  const ADDRESS = parseAddress('0xae3a8a1d877a446b22249d8676afeb16f056b44e')
```

Comment:
Same - why do you change address in tests?

Reply:
Ok, I've left change in only 1 test that failed with uppercase